schema

```diff
@@ -225,6 +225,11 @@
               "$ref": "#/definitions/meta"
             }
           },
+          "anyOf": [
+            {"required": ["data"]},
+            {"required": ["meta"]},
+            {"required": ["links"]},
```

Comment:
Please remove this trailing comma; it's [not valid JSON](http://www.2ality.com/2013/07/trailing-commas.html).